clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1119,11 +1120,11 @@ public Type type() {
         }
     }
 
-    protected List<String> headers() {
-        return Arrays.asList("Name", "Description", "Type", "Default", "Valid Values", "Importance");
+    public List<String> headers() {
```

Comment:
Seems this method and `getConfigValue` are only used in `toHtmlTable` and `toRst`. Would it make sense to move these implementations into `ConfigDefUtils` as well? I tried this out locally and it seems to work. Then we don't need to expose these methods and the definitions here just become stubs. For example:
```java
    public String toRst() {
        return ConfigDefUtils.toRst(this);
    }
```

Reply:
Unfortunately we can't completely remove `getConfigValue()` and `headers()` from `LogConfig` because `LogConfigDef` overrides these 2 methods. I feel like it make sense to keep these implementations in the class they belong.

I can still move the implementations of `toRst()`, `toEnrichedRst()` and `toHtmlTable()` into `ConfigDefUtils` and only keep stubs in `ConfigDef` so it's consistent with `toHtmlList()`.

Reply:
Man, that's annoying. Ok, I think increasing visibility of these methods is probably worse than the problem we were trying to solve. So how about we leave `toHtmlTable` untouched and we introduce a new `toHtml` implementation which outputs the config in list format? This is a little more consistent with the other generic methods such as `toRst` and it also gives us a little more flexibility to change the HTML output in the future. What do you think?

Reply:
I've tried a bunch of things and because we need to keep these 2 methods we can't cleanly hide the `toHtmlList()`.

I've removed `ConfigDefUtils` and just added `toHtmlList()` to `ConfigDef`. This is the simplest change.